cli/src/utils.ts

```diff
@@ -21,60 +18,21 @@ export const idOption = option({
   long: 'id'
 })
 
-export async function loadFile(filePath: string) {
-  const f = await openFile(filePath)
+export async function loadFile(source: string) {
+  const f = await openFile(source)
   return readFile(f)
 }
 
-export async function computeAdapterHash({
-  data,
-  verify
-}: {
-  data: IAdapter
-  verify?: boolean
-}): Promise<IAdapter> {
-  const input = JSON.parse(JSON.stringify(data))
-
-  // Don't use following properties in computation of hash
-  delete input.adapterHash
-
-  const hash = ethers.utils.keccak256(ethers.utils.toUtf8Bytes(JSON.stringify(input)))
-
-  if (verify && data.adapterHash != hash) {
-    throw new CliError(
-      CliErrorCode.UnmatchingHash,
-      `Hashes do not match!\nExpected ${hash}, received ${data.adapterHash}.`
-    )
-  } else {
-    data.adapterHash = hash
-    return data
-  }
+export async function loadJsonFromUrl(url: string) {
+  const res = await (await fetch(url, { method: 'Get' })).json()
```

Comment:
I guess this code works, but it is common to use upper case for HTTP request methods.

```suggestion
  return await (await fetch(url, { method: 'GET' })).json()
```